### ISSUE TITLE
Add alt text for images

### ISSUE THREAD

**Issue by nichgray** [CDRH/bandhistory_rails#44]
Alt text is missing for item images, gallery/lightbox images, and search result thumbnails. These are all overridden from datura and orchid so changes will need to be made here and in the data repo.

**Comment by nichgray** [comment]
This should partially be resolved with https://github.com/CDRH/bandhistory_rails/pull/42/changes/718b0a28770414c7318ba6cbd54dd99e592d1b4c and https://github.com/CDRH/data_bandhistory/pull/5/changes/c9d5c3b9839d5ac5d9e0980d42cea6c0178dc914, but the lightbox JS will probably also need to be adjusted.
